includes/controls/drop-shadow.php

```diff
@@ -0,0 +1,118 @@
+<?php
```

Comment:
Can it extend the box-shadow control?

Reply:
Hi @matipojo 
Apparently this task has been put on hold for now :-)